pytorch_lightning/accelerators/accelerator.py

```diff
@@ -480,7 +480,7 @@ def connect_precision_plugin(self, plugin: PrecisionPlugin) -> None:
         )
         self.setup_precision_plugin(plugin)
 
-    def save_checkpoint(self, checkpoint: Dict[str, Any], filepath) -> None:
+    def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: str) -> None:
```

Comment:
Nit: Could it be Path as well?

Reply:
@kaushikb11 Path isn't part of the typehint here: https://github.com/PyTorchLightning/pytorch-lightning/blob/1bd5f36a5bf02ba853c04de77647456c64784399/pytorch_lightning/plugins/training_type/training_type_plugin.py#L240

or here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/1bd5f36a5bf02ba853c04de77647456c64784399/pytorch_lightning/utilities/cloud_io.py#L45

which is why I didn't include it here